crates/viewer/src/camera.rs

```diff
@@ -8,14 +10,29 @@ enum Projection {
     Perspective,
 }
 
-pub struct Camera {
+pub struct OrbitCamera {
     projection: Projection,
     aspect_ratio: f32,
+    // Zoom factor used for orthographic projection.
+    zoom_factor: f32,
+    // The look-at target, in the center of the view.
+    target: Vec3,
+    // The radius of the orbit
+    radius: f32,
+    // The orientation of the camera around the target point
+    orientation: Quat,
 }
 
-impl Camera {
+impl OrbitCamera {
     pub fn new(width: u32, height: u32) -> Self {
-        Self { projection: Projection::Orthographic, aspect_ratio: width as f32 / height as f32 }
+        Self {
+            projection: Projection::Orthographic,
+            aspect_ratio: width as f32 / height as f32,
+            zoom_factor: 1.0,
+            target: vec3(0.0, 0.0, 0.0),
+            radius: 100.0,
+            orientation: Quat::IDENTITY,
```

Comment:
Here's my take on it:

```rust
let initial_camera_pos = vec3(2.0, -2.0, 1.0);

...

orientation: Quat::from_rotation_arc(
    Vec3::Z,
    initial_camera_pos.normalize(),
) * Quat::from_axis_angle(Vec3::Z, std::f32::consts::PI / 4.0),
```

Reply:
I made it parametric so you can just plug numbers in!